includes/core-changes/cryptography/5.0/cryptography-apis-not-supported-on-blazor-webassembly.md

```diff
@@ -0,0 +1,47 @@
+### System.Security.Cryptography APIs not supported on Blazor WebAssembly
+
+<xref:System.Security.Cryptography> APIs throw a <xref:System.PlatformNotSupportedException> at run time on the `browser` platform.
+
+#### Change description
+
+In previous .NET versions, you can use the <xref:System.Security.Cryptography> APIs in the Blazor WebAssembly configuration. In .NET 5.0 and later versions, these APIs throw a <xref:System.PlatformNotSupportedException> when invoked from projects that use the `Microsoft.NET.Sdk.BlazorWebAssembly` SDK or that include `browser` as a supported platform in the project file.
```

Comment:
@steveisok @pranavkm What's meant by "`browser` as a supported platform in the project file"? Does that refer to the following property in the project file of a Blazor WebAssembly project?

```xml
<RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
```

Reply:
https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-5-release-candidate-1/#browser-compatibility-analyzer-for-blazor-webassembly

Reply:
@gewarren I recommend clarifying this part by including `<SupportedPlatform Include="browser" />` (as mentioned in the linked blog post).